plugins/gssapi.c

```diff
@@ -2014,6 +2062,12 @@ static int gssapi_client_mech_step(void *conn_context,
     gss_cred_id_t client_creds = (gss_cred_id_t)params->gss_creds;
     gss_channel_bindings_t bindings = GSS_C_NO_CHANNEL_BINDINGS;
     struct gss_channel_bindings_struct cb = {0};
+    const char *service_principal_opt = NULL; /* The configured service principal name */
+#ifdef GSS_USE_LOAD_FROM
+    char* keytab = NULL; /* The configured credentials cache file path */
```

Comment:
The comment is wrong, a keytab is not a credential cache.

I am also not sure you should use "keytab" on the client side. GSSAPI distinguishes between "keytab" and "client_keytab" for good reasons.

Reply:
Removed all client changes. I have no immediate need for them.